cinch/roles/jenkins_master/tasks/iptables.yml

```diff
@@ -40,12 +40,7 @@
     jump: REDIRECT
     to_ports: "{{ https_enabled | ternary(8443, 8080) }}"
     state: present
-  register: real_address_rule
-
-- name: save iptables
-  shell: service iptables save
-  changed_when: false
-  when: forward_rule.changed or loopback_rule.changed or real_address_rule.changed
+  notify: save iptables
```

Comment:
This change breaks iptables. This command needs to be run at precisely this point, before iptables is starts below, or the old rules will be read back into the system. I'll be reverting this change in an upcoming PR.